Templates/hardware-readme-template.md

```diff
@@ -0,0 +1,79 @@
+<!--- This template should be used if the project contains hardware designs. If the project contains both hardware and software designs, please also refer to the software template and incorporate relevant sections. -->
+
```

Comment:
Name of the software template?  

We're saying here that we want only one readme, so would it be clearer/easier for contributors to have only one template and tell people to delete the irrelevant parts?

Reply:
as we expect many projects might be software-only or hardware-only, lets leave it as having two readmes for now.